js/idex.js

```diff
@@ -898,6 +899,7 @@ module.exports = class idex extends Exchange {
             amountEnum = 1;
             amount = this.safeFloat (params, 'quoteOrderQuantity');
         }
+        const clientOrderId = this.safeString (params, 'clientOrderId', '');
```

Comment:
We should not really set or send this value to an empty string, if the user didn't specify it. Can you add the conditional here?